h/static/scripts/widget-controller.js

```diff
@@ -204,9 +234,11 @@ module.exports = function WidgetController(
     searchClients.push(searchClient);
     searchClient.on('results', function (results) {
       if (annotationUI.hasSelectedAnnotations()) {
-        // Select appropriate tab - notes or annotations, for selection
-        annotationUI.selectTab(
-          tabTypeFromSelection(annotationUI.getState().selectedAnnotationMap, results));
+        // Select appropriate tab - notes, annotations or orphans, for selection
+        $rootScope.$on(events.ANNOTATIONS_SYNCED, function () {
```

Comment:
Given that we have a `selectTab` call in the _other_ listener for `events.ANNOTATIONS_SYNCED` below, we shouldn't have it here as well.

Something else that you should be aware of is that @finitud recently fixed a set of bugs where listeners were created with `$rootScope.$on` but not unregistered when the controller/directive was destroyed, which happens when the user logs out.

From now on, in controllers and directives, only `$scope.$on` should be used, rather than `$rootScope.$on`. @finitud can provide a more detailed explanation if you'd like.